http://lists.gnu.org/archive/html/bug-hurd/2011-03/msg00112.html

2011-03-29  Samuel Thibault  <samuel.thibault@ens-lyon.org>

        * Makerules ($(inst_libdir)/libc.so): Add -lmachuser -lhurduser to
        libc.so on GNU/Hurd.

No topgit branch. It's still unclear what we want to aim for.

---
 Makerules |    3 +++
 1 file changed, 3 insertions(+)
--- a/Makerules
+++ b/Makerules
@@ -988,6 +988,9 @@
 	      '$(libdir)/$(patsubst %,$(libtype.oS),$(libprefix)$(libc-name))'\
 	      ' AS_NEEDED (' $(slibdir)/$(rtld-installed-name) ') )' \
 	) > $@.new
+ifeq ($(patsubst gnu%,,$(config-os)),)
+	echo 'INPUT ( AS_NEEDED ( -lmachuser -lhurduser ) )' >> $@.new
+endif
 	mv -f $@.new $@
 
 endif
